container/libcontainer/handler.go

```diff
@@ -758,6 +757,86 @@ func (h *Handler) GetProcesses() ([]int, error) {
 	return pids, nil
 }
 
+// ResetWss to be run as gorutine for non blocking referenced bytes reset
+func (h *Handler) ResetWss(containerName string) error {
+	var err error = nil
+	if *referencedResetInterval == time.Duration(0) {
+		return err
+	}
+	tmp := *referencedResetInterval
+	time.Sleep(time.Duration(rand.Intn(int(tmp.Seconds()))))
```

Comment:
Why do you need a non-negative pseudo-random number here?

Reply:
As explained before, this is to spread the load